curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148e41c728f65138addf9cc.md

```diff
@@ -7,14 +7,24 @@ dashedName: step-66
 
 # --description--
 
-Setting the `scroll-behavior` to `smooth` is preferred by most users. However, some users find this to be too slow, and prefer to have the scrolling happen instantaneously.
+Certain types of motion-based animations can cause discomfort for some users. In particular, people with <dfn>vestibular</dfn> disorders have sensitivity to certain motion triggers.
 
-There exists a media rule to set CSS based on the user's browser settings. This media rule is called `prefers-reduced-motion`, and expects one of the following values:
+The `@media` at-rule has a media feature called `prefers-reduced-motion` to set CSS based on the user's preferences. It can take one of the following values:
 
-- `reduce`
-- `no-preference`
+-   `reduce`
+-   `no-preference`
 
-Wrap the appropriate rule within a `prefers-reduced-motion` media rule such that a `scroll-behavior` of `smooth` is only set if the user's browser setting is `no-preference`.
+```CSS
+@media (feature: value) {
```

Comment:
```suggestion
@media (conditions: value) {
```

Reply:
I used `feature` because it is a "media feature". I'm not sure if `conditions` is correct as that would be everything inside the parentheses not just the left-hand side of the colon.

I initially had `(rule:value)` but I thought that using `(feature:value)` would make it more clear that `prefers-reduced-motion` is the "media feature" and `no-preference` the "value" it can be set to.